charts/.ci/scripts/local-kube-score.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+helm template --values charts/.ci/values-kube-score.yaml charts/* -f charts/.ci/values-kube-score.yaml | kube-score score - \
```

Comment:
This is inconsistent with the `chart-testing` script we should either expect that the user has these tools installed (and should tell the user in the Readme what is required) or use only docker images.

Reply:
I would also prefer to have a `Makefile` target for that so we can simply run `make helm-lint` for example and call everything.